examples/animation-warps/index.html

```diff
@@ -18,11 +18,11 @@
         <!-- Green -->
         <a-entity>
           <a-animation attribute="scale" from="1 0 1" to="1 1 1" dur="2000" fill="both"></a-animation>
-          <a-cylinder position="0 2 0" radius="2" height="2" translate="0 -1 0" color="#7BC8A4" open-ended="true" side="double">
+          <a-cylinder position="0 2 0" radius="2" height="2" color="#7BC8A4" open-ended="true" side="double">
```

Comment:
A translation of 0 -1 0 seems pretty big given the size and the rest of the positions.